### ISSUE TITLE
Mentions is not defined

### ISSUE THREAD

**Issue by ryuuparattapon** [MRakox/discord-moderation#2]
I tried to mute but I got this error 
<img width="733" alt="error" src="https://user-images.githubusercontent.com/8351165/81481267-fd241880-9258-11ea-9198-699cc2453caf.PNG">

**Comment by MRakox** [comment]
Hi @ryuuparattapon,
Thank you for opening an issue and sorry for the late reply...
I made a mistake in the documentation you need to use `message.mentions.users.first()` instead of `mentions.users.first()`.

```js
bot.moderator.mute(message.guild.members.get(message.mentions.users.first().id), {
            time: ms(args[1]),
            reason: args.slice(2).join(" "),
            author: message.member,
            mutedRoleID: "689384052142243929"
        }).then((muteData) => {
            console.log(muteData); // {...} (id, type, author, reason and more)
        });
```

**Comment by MRakox** [comment]
**Fix in 1.0.5**
